Makefile

```diff
@@ -31,14 +31,14 @@ build-docs:
 	$(MAKE) -C docs html
 
 validate-docs: build-docs
-    # Required dependencies from docs/requirements.txt
+    # Required dependencies from docs-requirements.txt
 	python newsfragments/validate_files.py
 	towncrier --draft
 
 docs: build-docs validate-docs
 	readlink -f docs/build/html/index.html
 
-mac-docs: build-docs
+mac-docs: build-docs validate-docs
```

Comment:
It looks like validate includes build-docs: `validate-docs: build-docs` -- so this seems redundant

Reply:
@KPrasch and I chatted about it - leaving it as is